### PR TITLE
doc(panel): document meshPanel motto of the day config

### PR DESCRIPTION
**Pull request by JohannesRudolph**

### CODE REVIEW

File: docs/meshstack.configuration.md [hunk 1]
```diff
@@ -234,6 +234,53 @@ In order to use GCD as a lookup provider you need to provide these credentials:
 
 The GCD Service User needs read access to the [GCD Directory API](https://developers.google.com/admin-sdk/directory/v1/get-start/getting-started).
 
+### Motto of the Day
+
+Operators can configure an optional "motto of the day" to be displayed in meshPanel.
+This is useful to communicate important information such as newly available cloud plaforms or known issues to every user visiting meshPanel.
```
Comment (Jelledb) [inline docs/meshstack.configuration.md:240]:
```suggestion
This is useful to communicate important information such as newly available cloud platforms or known issues to every user visiting meshPanel.
```

File: docs/meshstack.configuration.md [hunk 2]
```diff
@@ -234,6 +234,53 @@ In order to use GCD as a lookup provider you need to provide these credentials:
 
 The GCD Service User needs read access to the [GCD Directory API](https://developers.google.com/admin-sdk/directory/v1/get-start/getting-started).
 
+### Motto of the Day
```
Comment (Jelledb) [inline docs/meshstack.configuration.md:237]: Isn't MOTD usually 'Message of the day' ? Or why did we go with Motto here

File: docs/meshstack.configuration.md [hunk 3]
```diff
@@ -234,6 +234,53 @@ In order to use GCD as a lookup provider you need to provide these credentials:
...
+    -}
+      { message : Text, startTime : Text, endTime : Text }
+```
+<!--Example-->
+```dhall
+let example
+    : Optional MottoOfTheDay
+    = Some
+        { message =
+            "The Likvid Bank Cloud Foundation Team whishes you a meshi <a href=\"https://en.wikipedia.org/wiki/Christmas\">Christmas</a>."
```
Comment (Jelledb) [inline docs/meshstack.configuration.md:273]: I like this example 🎄

**Review by Jelledb — CHANGES_REQUESTED** [review]

File: docs/meshstack.configuration.md (hunk 2 above)
Comment (JohannesRudolph) [reply docs/meshstack.configuration.md:237]: Yeah that sounds better I have to admit

File: docs/meshstack.configuration.md (hunk 3 above)
Comment (JohannesRudolph) [reply docs/meshstack.configuration.md:273]: yes 👍

**Review by Jelledb — APPROVED** [review]

**Pull request merged**